cluster-provision/k8s/1.24/k8s_provision.sh

```diff
@@ -160,6 +160,10 @@ envsubst < $kubeadm_raw > $kubeadm_manifest
 
 until ip address show dev eth0 | grep global | grep inet6; do sleep 1; done
 
+
+# prepull coredns:v1.8.6 and retag to expected tag
+podman pull registry.k8s.io/coredns/coredns:v1.8.6 && podman tag registry.k8s.io/coredns/coredns:v1.8.6 k8s.gcr.io/coredns:v1.8.6
```

Comment:
can we derive the tag from the manifests ? so once it is changed we dont need to update this code?
(unless it is just a WA and expected to be dropped soon)